front50-core/src/main/groovy/com/netflix/spinnaker/front50/model/delivery/EnvironmentMapping.java

```diff
@@ -0,0 +1,67 @@
+package com.netflix.spinnaker.front50.model.delivery;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+import java.util.List;
+import java.util.Map;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class EnvironmentMapping {
+  private String name;
+  private DeployWith deployWith;
+  private List<String> constraints;
```

Comment:
Does this need to be something more complex than a string? e.g. for the suggested "has to have been in test for > 30 minutes" constraint you might want to make that a generic rule with "test" and "30 minutes" being parameters.

Reply:
Yes, probably. Put in a string as a placeholder